--- config.inc.php	2017-07-23 15:44:07.946999730 +0100
+++ config.inc.php.patched	2017-07-23 15:46:23.094006808 +0100
@@ -172,7 +172,7 @@
 #    '/regular expression/' => '$PALANG key (optional: + parameter)',
     '/.{5}/'                => 'password_too_short 5',      # minimum length 5 characters
     '/([a-zA-Z].*){3}/'     => 'password_no_characters 3',  # must contain at least 3 characters
-    '/([0-9].*){2}/'        => 'password_no_digits 2',      # must contain at least 2 digits
+#    '/([0-9].*){2}/'        => 'password_no_digits 2',      # must contain at least 2 digits
 );
 
 // Generate Password
